subsys/bluetooth/host/gatt.c

```diff
@@ -3706,18 +3720,19 @@ static void gatt_discover_next(struct bt_conn *conn, uint16_t last_handle,
 	params->func(conn, NULL, params);
 }
 
-static void gatt_find_type_rsp(struct bt_conn *conn, uint8_t err,
+static void gatt_find_type_rsp(struct bt_conn *conn, int err,
 			       const void *pdu, uint16_t length,
 			       void *user_data)
 {
 	const struct bt_att_handle_group *rsp = pdu;
 	struct bt_gatt_discover_params *params = user_data;
 	uint8_t count;
 	uint16_t end_handle = 0U, start_handle;
+	uint8_t att_err = att_err_from_int(err);
 
-	LOG_DBG("err 0x%02x", err);
+	LOG_DBG("err 0x%02x", att_err);
```

Comment:
Suggest to either keep it as
```suggestion
	LOG_DBG("err 0x%02x", err);
```

or change it to 
```suggestion
	LOG_DBG("att_err 0x%02x", att_err);
```

Reply:
This applies multiple places